awx/api/metadata.py

```diff
@@ -141,7 +141,7 @@ def get_field_info(self, field):
                 pass
         if field.field_name == 'type':
             field_info['type'] = 'choice'
-        elif field.field_name in ('url', 'custom_virtualenv', 'token'):
+        elif field.field_name in ('url', 'token'): #'custom_virtualenv',
```

Comment:
I'd lean towards just deleting code rather than leaving behind stuff commented out. The diff will be more clear that way too.

Reply:
Not to mention, deleting code is _extremely_ cathartic **and** it aligns with an admirable personal goal, which is to leave this world having deleted more code than you added.

Reply:
do we want to delete it permanently though? or keep it while people are migrating?

Reply:
Followed up in DM but also posting here for posterity:

> We need to be able to tell on the backend if something had a custom venv so that we know if they need to specify a custom EE.
> We dont need to expose it in the API anymore, so it’s ok to go ahead and remove it from any serializers.

Reply:
Reading through this diff a bit more... We might end up needing _most_ of the code commented out here. We can definitely get rid of any references to custom venvs in the serializers / API, though. We just want to keep any resolving logic, so that we can prompt users in the appropriate situations.

Reply:
I pulled all changes out _except_ for the relevant model change. Since I'm not fully caught up on everything here I'm quite sure this change alone will not be enough and I'll need to hook in some more changes but this is a better, simpler place to start.